packages/data/lib/utils/const.dart

```diff
@@ -0,0 +1,9 @@
+class Config {
+  Config._();
+  static const sendTimeout = 5000;
+  static const receiveTimeout = 5000;
+  static const connectTimeout = 5000;
+  static const traktApiKey = 'trakt-api-key';
+  static const apiAnticipatedPath = 'https://api.trakt.tv/movies/anticipated';
```

Comment:
move base path https://api.trakt.tv to separate const and pass it when creating instance of Dio